x/auth/vesting/client/cli/tx.go

```diff
@@ -166,7 +261,63 @@ func NewMsgCreatePeriodicVestingAccountCmd() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().Bool(FlagMerge, false, "Merge new amount and schedule with existing true vesting account, if any")
+	cmd.Flags().String(FlagLockup, "", "path to file containing unlocking periods")
+	cmd.Flags().String(FlagVesting, "", "path to file containing vesting periods")
 	flags.AddTxFlagsToCmd(cmd)
+	return cmd
+}
+
+// NewMsgClawbackCmd returns a CLI command handler for creating a
+// MsgClawback transaction.
+func NewMsgClawbackCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "clawback [address]",
+		Short: "Transfer unvested amount out of a true vesting account.",
+		Long: `Must provide the funder address (--funder).
+		May provide a destination address (--dest), otherwise the coins return to the funder.
+		Delegated or undelegating staking tokens will be transferred in the delegated (undelegating) state.
+		The recipient is vulnerable to slashing, and must act to unbond the tokens if desired.`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			addr, err := sdk.AccAddressFromBech32(args[0])
+			if err != nil {
+				return err
+			}
 
+			funderString, _ := cmd.Flags().GetString(FlagFunder)
```

Comment:
Ugh - we've got some ugly choices here and I'd appreciate a better suggestion. At clawback time, we're transferring funds **from** the vesting account **to** a destination account, based on a message signed by the funder who was the "from" address when the account was created. So at clawback time, "from" is dangerously ambiguous. My proposed solution here is that at clawback time we say "funder" to avoid ambiguity, but at creation time we say "from" for consistency with the other commands to create vesting accounts.

Reply:
Turns out to be necessary to get the cli tests to work - at least without more poking into it. Done.